### DISCUSSION TITLE
how to validate a model resumed from a checkpoint and obtain accuracy without training?

### DISCUSSION THREAD

**Discussion by luminolx** [open-mmlab/mmclassification#1413] [General]
how to validate a model resumed from a checkpoint and obtain accuracy without training?

**Comment by Ezra-Yu** [comment]
Is it to test? refer to the [tutorial](https://mmclassification.readthedocs.io/en/dev-1.x/user_guides/train_test.html#test).

**Reply by luminolx** [reply]
I found it, but test.py can only output ['class_scores', 'pred_score', 'pred_label', 'pred_class'], NO accuracies.
I only want to get accuracies.
thx

**Reply by Ezra-Yu** [reply]
### display in CMD
``` 
python tools/test.py CONFIG CHECKPOINT
```

### dump to a file
``` 
python tools/test.py CONFIG CHECKPOINT --out results.pkl --out-item metrics
```

**Reply by luminolx** [reply]
found it, thx.
